smartsim/_core/launcher/dragon/dragonLauncher.py

```diff
@@ -85,21 +91,21 @@ class DragonLauncher(WLMLauncher):
 
     def __init__(self) -> None:
         super().__init__()
-        self._context = zmq.Context()
+        self._context = DRG_CTX
         self._timeout = CONFIG.dragon_server_timeout
         self._reconnect_timeout = CONFIG.dragon_server_reconnect_timeout
         self._startup_timeout = CONFIG.dragon_server_startup_timeout
         self._context.setsockopt(zmq.SNDTIMEO, value=self._timeout)
         self._context.setsockopt(zmq.RCVTIMEO, value=self._timeout)
         self._dragon_head_socket: t.Optional[zmq.Socket[t.Any]] = None
         self._dragon_head_process: t.Optional[subprocess.Popen[bytes]]
```

Comment:
Not directly related to this PR, but I thought I would tag it on: this attr will not be truly initialized until it is assigned some value
```suggestion
        self._dragon_head_process: t.Optional[subprocess.Popen[bytes]] = None
```

Reply:
Thanks!